### DISCUSSION TITLE
Filament tracker

### DISCUSSION THREAD

**Discussion by Digityle** [Arksine/moonraker#565] [Feature Requests]
I was curious if anyone had made any more progress on the filament manager @mateuszbrzezinski was working on, I'm in the process of learning python just to build a decent filament manager (I have a hx711 strain guage filament scale and rfid tagging with homeassistant) so, yeah was just wondering if I could begin building off of mateusz's code, I've messaged him multiple times with no response and it's obviously stalled out completely. Thanks again!

**Comment by Donkie** [comment]
Moonraker now supports interaction with Spoolman, so this discussion could be closed.

https://moonraker.readthedocs.io/en/latest/configuration/#spoolman
